### DISCUSSION TITLE
Kein Downlod vom WP All Import Add On möglich.

### DISCUSSION THREAD

**Discussion by Cadizan** [affiliatetheme/affiliatetheme#14] [Q&A] [answered]
Hallo. Ich möchte in unsere Website grandiosgross.de eine Produktsuche für extralange Bekleidung integrieren. 

Ich kann die notwendige Extension WP All Import Addon nicht downloaden.   

Auf der Website ist dies nur für registrierte Nutzer möglich. 
https://affiliatetheme.io/extensions/wp-all-import-addon/

Da ich das Theme jetzt als kostenlose, registrierungsfreie Version runtergeladen habe, habe ich keinen Zugriff.

Wäre super, wenn ich Tipp/Link bekommen könnte, zum download des AddOns. 

Danke!

**Comment by olaf-schmitz** [comment, marked as answer]
Hier ist der Download der Extension der Version 1.0 - ohne Gewähr, dass es mit der 2.0 und PHP 8.x funktioniert
[at-wp-all-import-addon.zip](https://github.com/affiliatetheme/affiliatetheme/files/11183989/at-wp-all-import-addon.zip)

**Comment by Cadizan** [comment]
Besten Dank Olaf für das Zip Olaf. Christian

 

Von: Olaf Schmitz ***@***.***>
Antworten an: affiliatetheme/affiliatetheme ***@***.***>
Datum: Samstag, 8. April 2023 um 21:37
An: affiliatetheme/affiliatetheme ***@***.***>
Cc: Christian ***@***.***>, Author ***@***.***>
Betreff: Re: [affiliatetheme/affiliatetheme] Kein Downlod vom WP All Import Add On möglich. (Discussion #14)

 

Hier ist der Download der Extension der Version 1.0 - ohne Gewähr, dass es mit der 2.0 und PHP 8.x funktioniert
at-wp-all-import-addon.zip

**Comment by markfrei** [comment]
Hallo, gibt es noch eine neuere Version als 1.0. ?
Wo kann man die bestehenden Plugins downloaden, nachdem affiliatetheme.io ja abgeschaltet wurde?

Vielen Dank
